src/librvnpal/inc/internal_win.h

```diff
@@ -0,0 +1,13 @@
+#ifndef INTERNALWIN_H
+#define INTERNALWIN_H
+
+#if defined(_WIN32)
+
+PRIVATE int32_t
+_write_file(void* handle, const void* buffer, int64_t size, int64_t offset, int32_t* detailed_error_code);
+
+PRIVATE int32_t
+_write_file_incrementally(void* handle, char** buffer, int64_t* size, int64_t* offset, int64_t increment_size, int32_t* detailed_error_code);
```

Comment:
what does this mean?

Reply:
This function writes the file in sections. Maybe `write_file_in_sections` will be more suitable?